services/main/Controllers/Waitlist.cs

```diff
@@ -0,0 +1,34 @@
+
+using main.DTOs;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+namespace main.Controllers;
+
+[ApiController]
+[Route("api/v1/waitlists")]
+public class WaitlistController : ControllerBase
+{
+    private readonly ILogger<WaitlistController> logger;
+    private readonly WaitlistService waitlistService;
+
+    public WaitlistController(ILogger<WaitlistController> logger, WaitlistService waitlistService)
+    {
+        this.logger = logger;
+        this.waitlistService = waitlistService;
+    }
+
+    [HttpPost("create")]
```

Comment:
```suggestion
    [HttpPost]
```

This will end up creating an endpoint like this `POST {domain}/api/v1/wailists`.

We're currently using the standard api design out there. Some examples

Create: `POST /api/v1/{resource}`, With body
Get All: `GET /api/v1/{resource}?limit=10&skip=1&...`
Get Single: `GET /api/v1/{resource}/:id`
Update: `PATCH /api/v1/{resource}/:id` - With body, of course
Delete: `DELETE /api/v1/{resource}/:id`